Dashboards/dashboard-SLA.json

```diff
@@ -0,0 +1,270 @@
+{
+	"id": "2b542873-793f-4eaa-8114-d855bb49cbc9",
```

Comment:
Lets give it a more appropriate id

Reply:
@Bars92 said it's a waste of time to change it and doesn't matter

Reply:
yes ignore.